services/core-web/src/components/noticeOfWork/applications/administrative/NOWApplicationAdministrative.js

```diff
@@ -17,18 +17,24 @@ const propTypes = {
   inspectors: CustomPropTypes.groupOptions.isRequired,
   setLeadInspectorPartyGuid: PropTypes.func.isRequired,
   handleUpdateLeadInspector: PropTypes.func.isRequired,
+  importNowSubmissionDocumentsJob: PropTypes.objectOf(PropTypes.any),
+  handleSaveNOWEdit: PropTypes.func.isRequired,
```

Comment:
Is this being used?

Reply:
importNowSubmissionDocumentsJob - is used in modal to display the import job status.
handleSaveNOWEdit - nope, it seems like something old and unused. I will delete it